extensions/devservices/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Dev Services"
+metadata:
+  guide: "https://quarkus.io/guides/dev-services"
+  status: "stable"
+  unlisted: true
+  config:
+    - "quarkus.devservices."
```

Comment:
I'm not sure to understand why we need a runtime extension here?

Do we have anything runtime for the other Dev UI things?

Reply:
Dev UI section is easier to add this way. Otherwise, I'd have to add it to vertx-http module

Reply:
@stuartwdouglas do we really want some Dev UI related components to leak into the runtime? From my point of view, it's really something we should avoid if we can but given you designed this thing, I'll let the decision to you.

Reply:
As I wrote to @geoand s earlier comment, only dependency devservices runtime module has is quarkus-arc.

Reply:
I have pushed a commit that removes the runtime dep, and moves the components into various other deployment modules. The 'Dev Services' link now lives on the first 'Configuration' card.